gifted_build/src/components/special_olympics.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import "../css/special_olympics.css"
+
+class Special_Olympics extends React.Component<{}, {}> {
+    render () {
+        return (
+            <div id="container">
+                <div id="special_olympics_img">
+                    <img src={require('../files/spolympics_logo.jpeg')} />
+                </div>
+                <div id="about_partnership_container">
```

Comment:
this is fine for now, but this shouldn't be a hardcoded entry, this should be variable to the data pulled from the database in order for more partners to be added.

Reply:
Will work on this once we have the database in the branch.